lib/tasks/work_tasks.rake

```diff
@@ -15,4 +15,25 @@ namespace :work do
     end
   end
 
+  # Usage: rake work:reset_word_counts[en]
```

Comment:
Would this be better in the after tasks rake file? That's where we usually put one-time use clean-up type things.